pkg/render/kube-controllers.go

```diff
@@ -54,55 +61,93 @@ func KubeControllers(
 	authentication *operator.Authentication,
 	enabledESOIDCWorkaround bool,
 	clusterDomain string,
-	esAdminSecret *v1.Secret,
+	kubeControllersUserSecret *v1.Secret,
+	kubeControllersVerificationSecret *v1.Secret,
+	esAdminUserSecret *v1.Secret,
 	metricsPort int,
 ) *kubeControllersComponent {
-	var elasticsearchUserSecret *v1.Secret
-	if esAdminSecret != nil {
-		elasticsearchUserSecret = &v1.Secret{
+	var kubeControllersSecureUserSecret *v1.Secret
+	if kubeControllersUserSecret == nil {
```

Comment:
I think it would make sense to get the logstorage controller to create this secret.

Thinking out loud here, this isn't really a secret for kube controllers, it's the "dummy" representation of the admin secret, so maybe it should be named accordingly.